views/partial/footer.ejs

```diff
@@ -1,2 +1,105 @@
+<body>
+
+    <!--The text and break tags below are placed just for testing purpose. They can be removed later-->
+    <h1>Scroll down to view the footer</h1><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+
+    <footer class="footer-distributed">
+
+        <div class="footer-left">
+    
+            <img src="public\img\pda-logo.png" alt="logo" class="logo">
+            
+            <h3>PDA</h3>
+    
+            <p class="footer-company-about">                
+                The Personality Development Association of MIT is an interdisciplinary
+    environment that allows similar yet different students to come together
+    and help each other develop soft skills and industrial maneuvers. Relevantly,
+    our motto is “Discover Yourself”.
+            </p>
+                        
+        </div>
+
+        <div class="footer-center-1">
+            <h3>Quick links</h3>
+            <ul class="footer-links">
+                <li>
+                    <a href="#">Home</a>
+                </li>
+
+                <li>
+                    <a href="#">Events</a>
+                </li>
+
+                <li>
+                    <a href="#">About</a>
+                </li>
+
+                <li>
+                    <a href="#">Login</a>
+                </li>
+
+                <li>
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+            
+        </div>
+    
+        <div class="footer-center-2">
+            
+            <h3>Get in Touch</h3>
+
+            <div>
+                <i class="fa fa-map-marker"></i>
+                <p>Madras Institute of Technology<span> Anna Univesity, Chrompet</span></p>
+            </div>
+    
+            <div>
+                <i class="fa fa-phone"></i>
+                <p>+91 97903 19116</p>
+            </div>
+    
+            <div>
+                <i class="fa fa-envelope"></i>
+                <p><a href="mailto:pdamit22@gmail.com">pdamit22@gmail.com</a></p>
+            </div>
+    
+        </div>
+    
+        <div class="footer-right">
+            
+            <h3>Follow us</h3>
+
+            <div class="footer-icons">
+    
+                <a href="https://www.facebook.com/mitpda"><i class="fa fa-facebook"></i></a>
+                <a href="https://www.instagram.com/pda_mit/"><i class="fa fa-instagram"></i></a>
+                <a href="https://www.linkedin.com/company/personality-development-association-mit/"><i class="fa fa-linkedin"></i></a>
+    
+            </div>
+
+            <a href="#" class="footer-sign-up">Sign up now!</a>
+            
+        </div>
+        <div class="top_scroll">
+            <span onclick="(function(){window.scrollTo({top: 0,left: 0,behavior: 'smooth'})})();"><i class="fa fa-arrow-circle-up"></i></span>
```

Comment:
Place this at the end 

![image](https://user-images.githubusercontent.com/53329034/147415705-cbad8705-b4c1-437f-87a7-fd53be29c764.png)